config/splat.pspeu.stwrp.yaml

```diff
@@ -62,73 +62,33 @@ segments:
-      - [0xEDD8, .data, wrp_psp/e_collect] # g_goldCollectTexts
-      - [0xEE50, .data, wrp_psp/e_collect] # c_GoldPrizes
-      - [0xEE80, .data, wrp_psp/e_collect] # c_HeartPrizes
-      - [0xEE88, .data, wrp_psp/e_collect] # g_ExplosionYVelocities
-      - [0xEEA0, .data, wrp_psp/e_collect] # D_80180ED8
-      - [0xEEB8, .data, wrp_psp/e_collect] # g_bigRedFireballAnim
-      - [0xEED8, .data, wrp_psp/e_collect] # D_80180F08
-      - [0xEF08, .data, wrp_psp/e_collect] # D_80180F38
-      - [0xEF40, .data, wrp_psp/e_collect] # D_80180F6C
-      - [0xEF48, .data, wrp_psp/e_collect] # g_ExplosionAnimations
-      - [0xEF60, .data, wrp_psp/e_collect] # g_RelicOrbTextBg1EY
-      - [0xEF70, .data, wrp_psp/e_collect] # g_RelicOrbTextBg1SY
-      - [0xEF80, .data, wrp_psp/e_collect] # g_RelicOrbTextBg2SY
-      - [0xEF90, .data, wrp_psp/e_collect] # g_RelicOrbTextBg2EY
-      - [0xEFA0, .data, wrp_psp/e_collect] # g_RelicOrbSparkleX
-      - [0xEFB0, .data, wrp_psp/e_collect] # g_RelicOrbSparkleY
-      - [0xEFC0, .data, wrp_psp/e_collect] # e_relic_orb obtained
-      - [0xF010, .data, wrp_psp/wrp_data_F010] # D_80180FF8
-      - [0xF020, .data, wrp_psp/wrp_data_F010] # D_80181008
-      - [0xF050, .data, wrp_psp/wrp_data_F010] # D_80181038
+      - [0xEA00, .data, wrp/e_red_door]
+      - [0xEA18, .data, wrp/st_update]
+      - [0xEA48, .data, wrp/e_particles]
+      - [0xEAC8, .data, wrp_psp/e_collect]
+      - [0xF010, .data, wrp_psp/e_misc]
       - [0xF060, .data, wrp/collision]
       - [0xF438, .data, wrp/e_room_fg]
-      - [0xF4D8, data]
-      - [0xF700, data] # D_psp_09246D80
-      - [0xF720, data] # D_psp_09246DA0
-      - [0xF740, .data, wrp_psp/wrp_data_F740] # D_psp_09246DC0
-      - [0xF790, .data, wrp_psp/wrp_data_F7B0] # WRP_RedDoorTiles
-      - [0xF7B0, .data, wrp_psp/wrp_data_F7B0] # D_psp_09246E30
-      - [0xF810, .data, wrp_psp/wrp_data_F810] # WRP_EInitBreakable
-      - [0xF820, .data, wrp_psp/wrp_data_F810] # g_EInitObtainable
-      - [0xF830, .data, wrp_psp/wrp_data_F810] # g_EInitParticle
-      - [0xF840, data] # D_psp_09246EC0
-      - [0xF850, data] # D_psp_09246ED0
-      - [0xF860, data] # D_80180488
-      - [0xF870, data] # g_EInitCommon
-      - [0xF880, data] # D_psp_09246F00
-      - [0xF890, data] # D_801804C4
-      - [0xF8A0, data] # D_psp_09246F20
-      - [0xF920, .data, wrp/e_breakable] # D_801805A8
-      - [0xF930, .data, wrp/e_breakable] # g_eBreakableAnimations
-      - [0xF950, .data, wrp/e_breakable] # g_eBreakableHitboxes
-      - [0xF970, data] # D_psp_09246FF0
+      - [0xF4D8, data] # WRP_spriteBanks
+      - [0xF5F8, data] # WRP_rooms_layers
+      - [0xF700, data] # WRP_cluts
+      - [0xF708, .data, wrp_psp/graphics_banks]
+      - [0xF790, .data, wrp_psp/e_init]
+      - [0xF8A0, data] # WRP_rooms
+      - [0xF920, .data, wrp/e_breakable]
+      - [0xF970, data] # [0xF970, cmpgfx, stage_title_jp, 128, 128, 4]
+      - [0xFCB8, data] # [0xFCB8, cmpgfx, stage_placeholder, 128, 128, 4]
```

Comment:
I was able to export these successfully to png, but re-linking them back into the build output caused a mismatch so I've commented this out for now. I might put up a WIP PR and see if I can get a hand on the linking side to understand where I'm going wrong.